docs/extended_api/synchronization_primitives/atomic_ref.md

```diff
@@ -0,0 +1,101 @@
+---
+grand_parent: Extended API
+parent: Synchronization Primitives
+nav_order: 1
+---
+
+# `cuda::atomic_ref`
+
+Defined in header `<cuda/atomic>`:
+
+```cuda
+template <typename T, cuda::thread_scope Scope = cuda::thread_scope_system>
+class cuda::atomic_ref;
+```
+
+The class template `cuda::atomic_ref` is an extended form of [`cuda::std::atomic_ref`]
+  that takes an additional [`cuda::thread_scope`] argument, defaulted to
+  `cuda::std::thread_scope_system`.
+It has the same interface and semantics as [`cuda::std::atomic_ref`], with the
+  following additional operations.
+This class additionally deviates from the standard by being backported to C++11.
+
+## Limitations
+
+`cuda::atomic_ref<T>` and `cuda::std::atomic_ref<T>` may only be instantiated with types sized between 4 and 8 bytes.
+
+## Atomic Fence Operations
+
+| [`cuda::atomic_thread_fence`] | Memory order and scope dependent fence synchronization primitive. `(function)` |
```

Comment:
Why is this here?

Reply:
Same reason as the atomic.md page, I've removed the section as I don't really have an opinion.